src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $cell-size);
+  justify-content: center;
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
```

Comment:
you cant use margins for blocks, you should make a container or use mixing for this purpose